pkg/flag/report_flags.go

```diff
@@ -71,6 +71,12 @@ var (
 		Value:      0,
 		Usage:      "specify exit code when any security issues are found",
 	}
+	ExitOnEoslFlag = Flag{
```

Comment:
nit

```suggestion
	ExitOnEOSLFlag = Flag{
```

cf. https://github.com/golang/go/wiki/CodeReviewComments#initialisms